client/src/utils/api-util.js

```diff
@@ -1,8 +1,78 @@
+import * as AWS from "aws-sdk";
+
+// Constant for the S3 Bucket name where the Monument images are stored
+const s3ImageBucketName = 'monument-images';
+// Constant for the S3 folder name where the Monument images are stored
+const s3ImageBucketFolderName = 'images/';
+
+/**
+ * Send a GET request to the specified url
+ * @param url - URL to send the GET to
+ */
 export async function get(url) {
     let error = null;
     let res = await fetch(url)
         .then(res => res.json())
         .catch(err => error = err);
     if (error || res.error) throw(error || res.error);
     else return res;
+}
+
+/**
+ * Send a POST request to the specified url with the specified data
+ * @param url - URL to send the POST to
+ * @param data - JSON data to send to the specified URL
+ */
+export async function post(url, data) {
```

Comment:
doesn't have to be done now, but we should probably abstract this into a private function that accepts the method as a param, sot hat we can easily do PUT and DELETE

Reply:
Sounds like a good idea. Don't feel like doing it now.